suites/quincy/rgw/tier-1_rgw_upgrade_5-3_to_6x_latest.yaml

```diff
@@ -0,0 +1,284 @@
+      name: download objects pre upgrade
+      polarion-id: CEPH-14237
+
+  - test:
+      name: Mbuckets_with_Nobjects with etag verification pre upgrade
+      desc: test etag verification
+      module: sanity_rgw.py
+      polarion-id: CEPH-83574871
+      config:
+        script-name: test_Mbuckets_with_Nobjects.py
+        config-file-name: test_Mbuckets_with_Nobjects_etag.yaml
+        timeout: 300
+
+  - test:
+      name: Dynamic Resharding tests pre upgrade
+      desc: Resharding test - dynamic
+      polarion-id: CEPH-83571740 # also applies to ceph-11479, ceph-11477
+      module: sanity_rgw.py
+      config:
+        script-name: test_dynamic_bucket_resharding.py
+        config-file-name: test_dynamic_resharding_without_bucket_delete.yaml
+        timeout: 500
+
+  - test:
+      name: Manual Resharding tests pre upgrade
+      desc: Resharding test - manual
+      polarion-id: CEPH-83571740
+      module: sanity_rgw.py
+      config:
+        script-name: test_dynamic_bucket_resharding.py
+        config-file-name: test_manual_resharding_without_bucket_delete.yaml
+        timeout: 500
+
+  - test:
+      name: Dynamic Resharding tests with version pre upgrade
+      desc: Resharding test - dynamic
+      polarion-id: CEPH-83571740
+      module: sanity_rgw.py
+      config:
+        script-name: test_dynamic_bucket_resharding.py
+        config-file-name: test_dynamic_resharding_with_version_without_bucket_delete.yaml
+        timeout: 500
+
+  - test:
+      config:
+        script-name: test_bucket_listing.py
+        config-file-name: test_bucket_listing_flat_ordered.yaml
+        timeout: 300
+      desc: test duration for ordered listing of bucket with top level objects
+      module: sanity_rgw.py
+      name: ordered listing of buckets pre upgrade
+      polarion-id: CEPH-83573545
+
+  - test:
+      config:
+        script-name: test_bucket_listing.py
+        config-file-name: test_bucket_listing_flat_unordered.yaml
+        timeout: 300
+      desc: test duration for unordered listing of buckets with top level objects
+      module: sanity_rgw.py
+      name: unordered listing of buckets pre upgrade
+      polarion-id: CEPH-83573545
+
+  - test:
+      config:
+        script-name: test_swift_basic_ops.py
+        config-file-name: test_swift_basic_ops.yaml
+        timeout: 300
+      desc: Test object operations with swift
+      module: sanity_rgw.py
+      name: swift basic operations pre upgrade
+      polarion-id: CEPH-11019
+
+  - test:
+      config:
+        script-name: test_bucket_listing.py
+        config-file-name: test_bucket_radoslist.yaml
+        timeout: 300
+      desc: Test bucket stats and radoslist on all buckets
+      module: sanity_rgw.py
+      name: test bucket stats and radoslist
+      polarion-id: CEPH-83574480
+
+# Object download test
+  - test:
+      name: Upgrade ceph
+      desc: Upgrade cluster to latest version
+      module: cephadm.test_cephadm_upgrade.py
```

Comment:
Just curious to know, conventionally we use **module: test_cephadm_upgrade.py** any reason why we are using **module: cephadm.test_cephadm_upgrade.py**